srtcore/core.cpp

```diff
@@ -3213,10 +2526,10 @@ bool CUDT::interpretSrtHandshake(const CHandShake& hs,
             const size_t bytelen = blocklen * sizeof(uint32_t);
             if (cmd == SRT_CMD_SID)
             {
-                if (!bytelen || bytelen > MAX_SID_LENGTH)
+                if (!bytelen || bytelen > CSrtConfig::MAX_SID_LENGTH)
                 {
                     LOGC(cnlog.Error,
-                         log << "interpretSrtHandshake: STREAMID length " << bytelen << " is 0 or > " << +MAX_SID_LENGTH
+                         log << "interpretSrtHandshake: STREAMID length " << bytelen << " is 0 or > " << (+CSrtConfig::MAX_SID_LENGTH)
```

Comment:
(parentheses) not necessary, arithmetic operators are above >> and << (unlike relative operators).